contrib/runners/winrm_runner/tests/unit/test_winrm_ps_script_runner.py

```diff
@@ -22,7 +22,7 @@
 from winrm_runner import winrm_ps_script_runner
 from winrm_runner.winrm_base import WinRmBaseRunner
 
-FIXTURES_PATH = os.path.join(os.path.dirname(__file__), "fixtures")
+from .fixtures import FIXTURES_PATH
```

Comment:
is the `.` necessary here?

Reply:
Alternatively we might be able to use this:
```suggestion
from tests.unit.fixtures import FIXTURES_PATH
```

Relative import was simpler. But I'm fine either way.